app/models/scheme.rb

```diff
@@ -212,6 +212,12 @@ def validate_confirmed
     end
   end
 
+  def validate_owning_organisation
+    unless owning_organisation.holds_own_stock?
+      errors.add(:owning_organisation_id, :does_not_own_stock, message: I18n.t("validations.scheme.owning_organisation.does_not_own_stock"))
```

Comment:
Do you need the second argument - `:does_not_own_stock` here?

Reply:
If I don't then I get a different validation error: "Enter the name of the organisation that owns the housing stock". (The validation I want to display is "Enter an organisation that owns housing stock"). Do you have a suggestion for a better way to deal with this? Thanks!